docs/examples/basic_conceptual_guide/Part_1-model_deployment/Readme.md

```diff
@@ -0,0 +1,293 @@
+```
+
+### Setting up the model repository
+
+A model repository is a filesystem based repository of models and configuration schema used by the Triton Inference Server. While a server can use one or more model repositories, for the purposes of simplicity, this explanation only uses a single repository. The general structure of a model repository is as follows:
+```
+<model-repository>/
+  <model-name>/
+    [config.pbtxt]
+    [<output-labels-file> ...]
+    <version>/
+      <model-definition-file>
+    <version>/
+      <model-definition-file>
+    ...
+  <model-name>/
+    [config.pbtxt]
+    [<output-labels-file> ...]
+    <version>/
+      <model-definition-file>
+    <version>/
+      <model-definition-file>
+    ...
+  ...
+```
+
+There are three important components to be discussed from the above structure:
+* `model name`: The identifying name for the model.
+* `config.pbtxt`: For each model, users need to define the model configuration. This configuration, at minimum, needs to define the name of the platform, and the name, shape, and datatype of the input and output layers.
+* `version`: versioning allows for multiple versions of the same model to be available for use depending on the policy selected.
+
+For this example the model repository structure would need to be set up in the following manner:
+```
+model_repository
+|
+|-- text_detection
+|   |
+|   |-- config.pbtxt
+|   |-- 1
+|       |
+|       |-- model.onnx
+|
+|-- text_recognition
+    |
+    |-- config.pbtxt
+    |-- 1
+        |
+        |-- model.onnx
+```
+
+With the models and the filestructure ready, the only point of dicussion remaining are the `config.pbtxt` files. As an example, consider the model configuration for the `EAST text detection` model. This is an ONNX model that has one `input` and two `output` layers.
+
+```
+name: "text_detection"
+platform: "onnxruntime_onnx"
+max_batch_size : 256
+input [
+  {
+    name: "input_images:0"
+    data_type: TYPE_FP32
+    dims: [ -1, -1, -1, 3 ]
+  }
+]
+output [
+  {
+    name: "feature_fusion/Conv_7/Sigmoid:0"
+    data_type: TYPE_FP32
+    dims: [ -1, -1, -1, 1 ]
+  }
+]
+output [
+  {
+    name: "feature_fusion/concat_3:0"
+    data_type: TYPE_FP32
+    dims: [ -1, -1, -1, 5 ]
+  }
+]
+```
+* **name:** "name" is an optional field, the value of which should match the name of the directory of the model.
+* **platform:** This field indicates which framework is being used to run the model. Triton supports a wide variety of backends like TensorFlow, Pytorch, Python, ONNX and more. For a complete list of field selection refer to [these comments](https://github.com/triton-inference-server/common/blob/main/protobuf/model_config.proto#L1791)
+* **max_batch_size:** As the name implies, this field defines the maximum batch size that the model can support.
+* **input and output layers**: The input and output layers specify the name, shape datatype and more, while providing operations like reshaping and more.
```

Comment:
Nit (comma): shape, datatype, and more.

The subordinate clause "while providing operations like reshaping and more..." may be confusing for folks. You could probably change it to be a second sentence that specifies that you could use this field as well to reshape inputs/outputs (with link to documentation) and allow ragged batching (with link to documentation). I think those are the only two additional options for these fields right now. Being more specific here could ease the cognitive load for users reading this for the first itme.

Reply:
You could also limit this description to its core function of providing the name, shape, and datatype of inputs/outputs to reduce the amount of information users receive at once. Users can discover reshape and ragged batching once they need it later on.